src/VisualStudio/Core/Def/Implementation/AbstractVsTextViewFilter`2.cs

```diff
@@ -47,18 +48,24 @@ int IVsTextViewFilter.GetDataTipText(TextSpan[] pSpan, out string pbstrText)
         }
 
         protected virtual int GetDataTipTextImpl(TextSpan[] pSpan, out string pbstrText)
+        {
+            var subjectBuffer = WpfTextView.GetBufferContainingCaret();
```

Comment:
>WpfTextView.GetBufferContainingCaret(); [](start = 32, length = 39)

So we are still usign the subject buffer of the caret, not the subject buffer that is hovered over?